latch/ldata/_transfer/utils.py

```diff
@@ -1,5 +1,91 @@
 import os
-from typing import List
+import time
+from enum import Enum
+from typing import Any, Callable, Dict, Optional
+
+from gql.gql import DocumentNode
+from gql.transport.exceptions import TransportClosed, TransportServerError
+from latch_sdk_gql import JsonValue
+from latch_sdk_gql.execute import execute
+
+from latch_cli import tinyrequests
+
+
+class HTTPMethod(Enum):
+    post = "post"
+    put = "put"
+    get = "get"
+
+
+req_method_map: Dict[HTTPMethod, Callable] = {
```

Comment:
unnecessary, can just use `tinyrequests.request` and pass in the method as the first arg